packages/config/config/devices/0x0287/iblindsv3.json

```diff
@@ -98,14 +100,78 @@
 				}
 			]
 		},
-		"6": {
-			"label": "Speed",
+		"6":{
+			"label": "Movement Duration",
 			"description": "How fast iblinds will close in seconds",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
 			"defaultValue": 0,
-			"unsigned": true
+			"unsigned": true,
+			"unit": "seconds"
+		},
+		"7":{
+			"$if": "firmwareVersion >= 3.6",
+			"label": "Remote Calibration",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"writeOnly": true,
+			"options": [
+				{
+					"label": "Idle",
+					"value": 0
+				},
+				{
+					"label": "Start calibration",
+					"value": 1
+				}
+			]
+		},
+		"8":{
+			"$if": "firmwareVersion >= 3.7",
+			"label": "Minimum Tilt Level",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 25,
+			"defaultValue": 0,
+			"allowManualEntry": true,
+			"writeOnly": true,
+			"unit":"Tilt %"
+		},
+		"9":{
+			"$if": "firmwareVersion >= 3.7",
+			"label": "Maximum Tilt Level",
+			"valueSize": 1,
+			"minValue": 75,
+			"maxValue": 99,
+			"defaultValue": 99,
+			"allowManualEntry": true,
+			"writeOnly": true,
+			"unit":"Tilt %"
+		},
+		"10":{
+			"$if": "firmwareVersion >= 3.7",
+			"label": "Remap 99",
+			"description":"Remap ON to 0x63 instead of 0xFF. Use if ON sends tilt all the way up instead of 50% or default ON.",
```

Comment:
Which command(s) does this apply to? I'd like to make this label/description a bit more concise.

Reply:
This applies to the Basic Switch (ON/OFF), and remaps 0X63 (99%) to Default On (Parameter 4) Value  (Default 50%). Some Z-Wave controllers send (0x63, instead of 0XFF for ON and this causes blind when commanded to open/ON to tilt to the maximum end point 99%. 
The description I wrote is actually backwards,
Remap Default ON to 0xFF instead of 0x63